projects/packages/blaze/src/class-blaze.php

```diff
@@ -78,15 +78,16 @@ public static function add_post_links_actions() {
 
 	/**
 	 * Is the wp-admin Dashboard enabled?
-	 * That dashboard is not available or necessary on WordPress.com sites.
+	 * That dashboard is not available or necessary on WordPress.com sites when the nav redesign is disabled.
 	 *
 	 * @return bool
 	 */
 	public static function is_dashboard_enabled() {
-		$is_dashboard_enabled = true;
+		$is_dashboard_enabled          = true;
+		$wpcom_is_nav_redesign_enabled = function_exists( 'wpcom_is_nav_redesign_enabled' ) && wpcom_is_nav_redesign_enabled();
```

Comment:
Should this be checking classic view site option instead of nav redesign? Or both maybe? e.g.

```
$wpcom_is_nav_redesign_enabled = function_exists( 'wpcom_is_nav_redesign_enabled' ) && 
    wpcom_is_nav_redesign_enabled() &&
    'wp-admin' === get_option( 'wpcom_admin_interface' );
```

I very well could be over thinking this. I'm wondering, if we remove the wpcom_is_nav_redesign enabled flag later on when we're happy with the release and are cleaning up, should this keep working or should it be limited to classic view only?

I'm aware we're checking the option inside the function as well.

Maybe we should leave it as is and be careful when we remove these function calls later? cc @fushar you like this logic.

Reply:
> I'm aware we're checking the option inside the function as well.

Yep. I think we can be careful when we enable the flags 🙂

Reply:
This is an interesting question. The flag is already used in multiple places. E.g. here as well https://github.com/Automattic/wpcomsh/pull/1688.

I guess, we should think about whether to enable the said features for classic only or not, later when we actually do the cleanup, as we will touch the flag anyway. For now I think it's too early to spend some thinking on it.

Reply:
I expected this to check the classic view site option and be for proxied users only, because that's where we disabled nav unification: https://github.com/Automattic/jetpack/pull/35642 Is there value in this change outside of that view?

Reply:
Yep. This is how [wpcom_is_nav_redesign_enabled](https://github.com/Automattic/wpcomsh/blob/9855665b64bd85776ce8a8fe6908cb8b48fac463/feature-plugins/nav-redesign.php#L13) function works 🙂

Reply:
Raised this as an issue to discuss more over in https://github.com/Automattic/dotcom-forge/issues/5712